File: test/templates/blank.tpl.go
package main

import (
	"fmt"

	"github.com/kuzzleio/sdk-go/connection/websocket"
	"github.com/kuzzleio/sdk-go/kuzzle"
	"github.com/kuzzleio/sdk-go/types"
)

func main() {
	[snippet-code]
	k.Server.Now(nil)
	fmt.Println("Everything is ok")
}

File: test/templates/blank.tpl.js
// load the Kuzzle SDK module
const
  {
    Kuzzle,
    Websocket,
    Http
  } = require('kuzzle-sdk');

const kuzzle = new Kuzzle(
  new Websocket({ host: 'kuzzle' })
);

[snippet-code]
console.log('Everything is ok');
